content/applications/websites/livechat/reports.rst

```diff
@@ -0,0 +1,176 @@
+=======
+Reports
+=======
+
+Odoo *Live Chat* includes several reports that allow for the monitoring of operator performance and
+the identification of trends in customer conversations.
+
+Available reports
+=================
+
+The following reports are included in the *Live Chat* app:
+
+- :ref:`Sessions History <livechat/sessions-history>`
+- :ref:`Session Statistics <livechat/session-statistics>`
+- :ref:`Operator Analysis <livechat/operator-analysis>`
+
+.. note::
+   The *Live Chat Ratings Report* can also be accessed through the :guilabel:`Report` menu. For more
+   information on this report, and on the *Live Chat* rating process, see :doc:`Live Chat Ratings
+   <../livechat/ratings>`.
+
+To access the available reports, navigate to the :menuselection:`Live Chat app --> Report menu`.
+
+.. _livechat/sessions-history:
+
+Sessions History
+----------------
+
+The *Sessions History* report displays an overview of live chat sessions, including participant
+names, session dates, and the duration of the session. It also provides access to the complete
+transcripts of live chat sessions, which open in the *Discuss* app.
+
+.. image:: reports/sessions-history.png
+   :align: center
+   :alt: Example of the Sessions History report from the Live Chat application.
+
+The information in this report can be exported or inserted into a spreadsheet. Click the
+:guilabel:`⚙️ (gear)` icon at the end of the words :guilabel:`History` in the top-left corner. This
+reveals a drop-down menu. Click :guilabel:`Export All` to export all sessions to a spreadsheet, or
+:guilabel:`Spreadsheet` to insert the information in a new or existing spreadsheet.
+
+To export only select sessions, first select the sessions to be exported from the list. Then, click
+the checkbox to the left of each individual session to select it. Click the :guilabel:`⚙️ (gear)
+Actions` icon in the top-center of the page, and click :guilabel:`Export`, or
+:guilabel:`Spreadsheet`.
+
+To view the transcript of an individual conversation, select it from the list and click. The page
+conversation opens in the *Discuss* app.
+
+In the *Discuss* app, the conversation view displays the entire transcript of the conversation. At
+the top of the conversation is a list of the web pages the visitor browsed before beginning their
+chat session, with time stamps. If the visitor left a rating, it is included at the end of the
+transcript.
+
+.. image:: reports/chat-transcript.png
```

Comment:
Optional- but is there a chat record of a happy exchange? Just to keep things light and not focus on upset customers =D  Like "I am missing a screw for my table" "Ok, we're mailing it to you right away" "5 stars", or something similar.